### PR TITLE
Fix frf 215 removed broken aria controls in navigation menu

### PR DESCRIPTION
**Pull request by ThomasJackSmolen**
Removed the broken aria-controls label in Header.tsx to remove this error from all pages.
Removed the test for this in the Home.spec.tsx file.

### CODE REVIEW

**Comment by Jaffacakes82** [comment]
@kevinofficer I'm not sure if this PR is merging into the right branch? Please can you check.

**Comment by kevinofficer** [comment]
@Jaffacakes82 yep all good, I've got the apprentices merging into temp once approved I'm merging them into main

**Comment by Jaffacakes82** [comment]
So can this be closed without merging?

**Review by kevinofficer — CHANGES_REQUESTED** [review]

File: src/mocks/homepage/success.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
         "serviceDescriptionFollowUp": "Collecting participant specific events and outcomes by matching to existing data sets.",
         "signPostDescription1": "Need help finding appropriate data services, or any other part of the UK journey to getting your study started? The UK offers multiple services and teams of professionals who can support you.",
         "signPostDescription2": "If you provide services that support Find, Recruit or Follow-up activties and would be interested in inclusion within this website, we would love to hear from you.",
-        "metaTitle": "Homepage Title",
```
Comment (kevinofficer) [inline src/mocks/homepage/success.json:65]: Add this title back into the file, and run the unit tests. 
You can run the unit tests with the command line: npm run test

**Review by kevinofficer — APPROVED** [review]

**Pull request merged**